hendrix/resources.py

```diff
@@ -215,9 +215,6 @@ def DjangoStaticResource(path, rel_url='static'):
     rel_url = rel_url.strip('/')
     StaticFilesResource = MediaResource(path)
     StaticFilesResource.namespace = rel_url
-    chalk.green(
```

Comment:
The emission is too verbose on larger projects. We can do better. Perhaps by adding a final statement after the statics are loaded (pointing to a log of the complete statics listing)